charactersheet/charactersheet/templates/dm/encounter.tmpl.html

```diff
@@ -0,0 +1,95 @@
+<div class="row">
+  <div class="col-xs-12">
+    <div class="h2">
+      Encounters<br/>
+      <small>Create and manage events and sections of your campaign. Group encounters under a common topic, or...</small>
+    </div>
+    <div class="row">
+      <div class="col-xs-12 col-sm-4">
+        <div class="panel panel-default">
+          <div class="panel-body">
+            <div class="panel-heading">
+              <div class="h4">
+                Encounters
+                <div class="pull-right">
+                  <span class="glyphicon glyphicon-plus" data-bind="click: openAddModal"></span>
+                </div>
+              </div>
+            </div>
+            <encounter-list params="encounters: encounters,
+              selectedEncounter: selectedEncounter, ondelete: deleteEncounter,
+              onadd: openAddModalWithParent">
+            </encounter-list>
+          </div>
+        </div>
+      </div>
+      <div class="col-xs-12 col-sm-8">
+        <!-- ko if: encounterDetailViewModel() -->
+        <div data-bind="template: {
+          name: 'encounter_detail.tmpl',
+          templateUrl: 'templates/dm',
+          data: encounterDetailViewModel
+        }"></div>
+        <!-- /ko -->
+      </div>
+    </div>
+  </div>
+</div>
+
+<!-- Add Modal -->
+<div class="modal fade" tabindex="-1" role="dialog"
```

Comment:
This is currently not do-able with the current design. This was something that I wanted to look into, for an improvement PR.